### PR TITLE
Updating GPG keys needed for install binaries. Fixes #12913.

### LINKED ISSUE #12913: Vagrantfile shell provisioning fails on GPG Key insertion
## Steps to reproduce

How'd you do it?

1. Clone Repo
2. Install Vagrant
3. Run `vagrant up`

## Expected behavior

Vagrant should provision the base box successfully for the environment to be useable.

## Current behavior

Vagrant provisioning fails with the following error:

```
    default: Downloading https://github.com/rvm/rvm/archive/1.29.9.tar.gz
    default: Downloading https://github.com/rvm/rvm/releases/download/1.29.9/1.29.9.tar.gz.asc
    default: gpg: Signature made Wed 10 Jul 2019 08:31:02 AM UTC using RSA key ID 39499BDB
    default: gpg: Can't check signature: No public key
    default: GPG signature verification failed for '/home/vagrant/.rvm/archives/rvm-1.29.9.tgz' - 'https://github.com/rvm/rvm/releases/download/1.29.9/1.29.9.tar.gz.asc'! Try to install GPG v2 and then fetch the public key:
    default:
    default:     gpg2 --keyserver hkp://pool.sks-keyservers.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
    default:
    default: or if it fails:
    default:
    default:     command curl -sSL https://rvm.io/mpapis.asc | gpg2 --import -
    default:     command curl -sSL https://rvm.io/pkuczynski.asc | gpg2 --import -
    default:
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
went wrong.
```

### Metasploit version

Latest from /master

### I installed Metasploit with:
- [ ] Kali package via apt
- [ ] Omnibus installer (nightly)
- [ ] Commercial/Community installer (from http://www.rapid7.com/products/metasploit/download.jsp)
- [x] Source install (please specify ruby version)

### OS

Ubuntu 16.04 Xenial 64 (Vagrant)
Linux version 4.4.0-173-generic

### PR DESCRIPTION
**Pull request by GuillaumeCleme**
Updated Vagrantfile shell provisioning commands to install updated GPG keys required for binaries install.

## Verification

- [x] Install Vagrant
- [x] Run `vagrant up`
- [x] **Verify** that the Vagrant provisioning completes without any error

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -28,7 +28,7 @@ Vagrant.configure(2) do |config|
     config.vm.provision "shell", inline: step
   end
 
-  [ "gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3",
+  [ "gpg2 --keyserver hkp://pool.sks-keyservers.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB",
```
Comment (acammack-r7) [inline Vagrantfile]:
```suggestion
  [ "gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB",
```
I don't think we want to update the GPG version or keyserver at this time.

**Review by acammack-r7 — CHANGES_REQUESTED** [review]
Thanks for keeping this up-to-date! I think we'll only want to add the addition RVM key and not either of the other changes at this time, though.

File: Vagrantfile (hunk 1 above)
Comment (GuillaumeCleme) [reply Vagrantfile]: Implemented suggested changes and tested successfully.

**Comment by busterb** [comment]
LGTM, thanks @GuillaumeCleme

**Pull request merged**